packages/core/core/test/TargetResolver.test.js

```diff
@@ -547,6 +547,44 @@ describe('TargetResolver', () => {
     );
   });
 
+  it('generates a default target in server mode', async () => {
```

Comment:
```suggestion
  it('generates a default target in serve mode', async () => {
```

there's a typo in the title, I think